heap.c

```diff
@@ -100,6 +110,20 @@ siftup(struct heap *h, size_t pos)
 }
 
 void
+heap_heapify(struct heap *h)
+{
+	for (ssize_t i=ptrvec_size(h->vec)/2-1; i>=0; i--) {
+		siftup(h, i);
```

Comment:
I was mainly wondering if the algorithm can fail, are we ignoring the error return? If it's just a matter of propagating the return value or not, and the algorithm always succeeds, then it's OK.